doc/devices/qiskit.rst

```diff
@@ -0,0 +1,47 @@
+Qiskit device
+=============
+The ``orquestra.qiskit`` device provided by the PennyLane-Orquestra plugin allows you to use PennyLane
+to deploy and run your quantum machine learning models on the backends and simulators provided
+by `Qiskit Aer <https://qiskit.org/aer/>`_.
+
+You can instantiate a ``'orquestra.qiskit'`` device for PennyLane with:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device('orquestra.qiskit', wires=2)
+
+This device can then be used just like other devices for the definition and evaluation of QNodes within PennyLane.
+A simple quantum function that returns the expectation value of a measurement and depends on three classical input
+parameters would look like:
+
+.. code-block:: python
+
+    @qml.qnode(dev)
+    def circuit(x, y, z):
+        qml.RZ(z, wires=[0])
+        qml.RY(y, wires=[0])
+        qml.RX(x, wires=[0])
+        qml.CNOT(wires=[0, 1])
+        return qml.expval(qml.PauliZ(wires=1))
+
+You can then execute the circuit like any other function to get the quantum mechanical expectation value.
+
+.. code-block:: python
+
+    circuit(0.2, 0.1, 0.3)
+
+Backends
+~~~~~~~~
+
+By default, the ``orquestra.qiskit`` device uses the noisy ``'qasm_simulator'``
+backend, but this may be changed to ``'statevector_simulator'``.  For more
```

Comment:
(minor) I read this as the default may be changed to `statevector_simulator` in the future, rather than that the user can change it (which I assume is what you mean here). I think using "can" might work better here. :slightly_smiling_face: 
```suggestion
backend, but this can be changed to ``'statevector_simulator'``.  For more
```

Reply:
Fair point, thanks!